stencils/pace/stencils/testing/test_translate.py

```diff
@@ -378,7 +378,14 @@ def test_parallel_savepoint(
         ref_data[varname] = []
         new_ref_data = all_ref_data[varname]
         if hasattr(case.testobj, "subset_output"):
-            new_ref_data = case.testobj.subset_output(varname, new_ref_data)
+            do_not_subset_tracers = getattr(
+                case.testobj, "_do_not_subset_tracers", None
+            )
+            if do_not_subset_tracers:
+                if varname in case.testobj._tracer_names:
+                    pass
+            else:
+                new_ref_data = case.testobj.subset_output(varname, new_ref_data)
```

Comment:
You should be able to avoid any changes to this fixture by instead modifying FVDynamics.subset_output instead. I'd like to avoid adding more coupling between the fixtures and the translate classes.